mscore/fotomode.cpp

```diff
@@ -591,7 +591,6 @@ void ScoreView::fotoDragDrop(QMouseEvent*)
       QRectF r(_foto->bbox());
 
       QTemporaryFile tf(QDir::tempPath() + QString("/imgXXXXXX.svg"));
```

Comment:
Is this tested? because if the file is removed at the end of this method, it will be missing when dropping no?

Reply:
Unaddressed review?

Reply:
No I did add that TODO comment, in the next line